### ISSUE TITLE
Добавить динамику развития болезни по странам

### ISSUE THREAD

**Issue by BupycHuk** [Entea/coronavirus-kyrgyzcha#12]
Сделать отдельную страницу для каждой страны где будет выводиться динамика развития болезни по дням.

**Comment by adilets** [comment]
А у API есть возможность дать данные по дням ? Я что то не нашел

**Comment by Entea** [comment]
@adilets да, см https://coronavirus-tracker-api.herokuapp.com/v2/locations?timelines=1 в backend/index.php

**Comment by tabekg** [comment]
Написал несколько недель назад [CoronaKG.web.app](https://coronakg.web.app)  динамически цифры обновляется

**Comment by BupycHuk** [comment]
@tabekg это совершенно о другом.
